microraiden/microraiden/client/client.py

```diff
@@ -42,7 +42,11 @@ def __init__(
         # Create web3 context if none is provided, either by using the proxies' context or creating
         # a new one.
         if not web3:
-            web3 = Web3(HTTPProvider())
+            web3 = Web3(HTTPProvider('http://localhost:8545'))
```

Comment:
Endpoint URL should be probably configurable.

Reply:
It is, by passing a different `Web3` object. Although @loredanacirstea's suggestion of using a config default might be more elegant.